.github/ISSUE_TEMPLATE/discussion.yaml

```diff
@@ -0,0 +1,34 @@
+name: Discussion
+description: Open up a new discussion about anything related to the project.
+labels: "Discussion"
+body:
+- type: checkboxes
+  attributes:
+    label: Guidelines
+    description: Please ensure you've completed all of the following.
+    options:
+        - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a discussion that matches the one i want to file, without success.
+          required: true
+- type: checkboxes
```

Comment:
Is it possible to make at least one checkbox required?

Reply:
I don't think so (maybe use a drop down list instead?)

Reply:
How would that work? What if its applicable to multiple subjects?

Reply:
Adding "multiple:true" to the attributes would allow multiple subjects to be selected

Ex:
```
  - type: dropdown
    id: browsers
    attributes:
      label: What browsers are you seeing the problem on?
      multiple: true
      options:
        - Firefox
        - Chrome
        - Safari
        - Microsoft Edge
```
Reference: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms